Riot/Modules/Threads/Beta/ThreadsBetaViewController.swift

```diff
@@ -0,0 +1,178 @@
+// 
+// Copyright 2022 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+
+class ThreadsBetaViewController: UIViewController {
+
+    // MARK: Constants
+
+    private enum Constants {
+        static let learnMoreLink = "https://element.io"
```

Comment:
Is this a temporary link? Or do we just link to the homepage?

Reply:
We were waiting for the final link, updated.